### ISSUE TITLE
Activity stream actions not overridable

### ISSUE THREAD

**Issue by amercader** [ckan/ckan#4684]
Some calls to action functions in `action/get.py` call the functions directly instead of using `get_action`, meaning they can't be overridden from extensions.

**Comment by amercader** [comment]
Just noticed that after #4627 this is no longer relevant in master. I'll target 2.8 for this.
